Description: Correct the FSF adress in some files
 In the upstream tar ball the following files
  - compat/avisynth/avisynth_c.h
  - compat/avisynth/avxsynth_c.h
 refer to an old FSF address.
 I updated that to: 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA

Author: Andreas Cadhalpun <Andreas.Cadhalpun@googlemail.com>
Forwarded: <https://ffmpeg.org/pipermail/ffmpeg-devel/2014-March/155297.html>
Applied-Upstream: <https://ffmpeg.org/pipermail/ffmpeg-devel/2014-March/155304.html>
Last-Update: <2014-03-24>

--- ffmpeg-2.2.orig/compat/avisynth/avisynth_c.h
+++ ffmpeg-2.2/compat/avisynth/avisynth_c.h
@@ -13,7 +13,8 @@
 //
 // You should have received a copy of the GNU General Public License
 // along with this program; if not, write to the Free Software
-// Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA, or visit
+// Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston,
+// MA 02110-1301 USA, or visit
 // http://www.gnu.org/copyleft/gpl.html .
 //
 // As a special exception, I give you permission to link to the
--- ffmpeg-2.2.orig/compat/avisynth/avxsynth_c.h
+++ ffmpeg-2.2/compat/avisynth/avxsynth_c.h
@@ -13,7 +13,8 @@
 //
 // You should have received a copy of the GNU General Public License
 // along with this program; if not, write to the Free Software
-// Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA, or visit
+// Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston,
+// MA 02110-1301 USA, or visit
 // http://www.gnu.org/copyleft/gpl.html .
 //
 // As a special exception, I give you permission to link to the
